<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="clsAddTab">
       <attribute name="title">
        <string>학급추가</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="fileUpdBtn">
          <property name="text">
           <string>파일 불러오기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="stdListWidget"/>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="grdListComboBox">
             <property name="currentText">
              <string>1</string>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="grdListLabel">
             <property name="text">
              <string>학년</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="clsListComboBox">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>17</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="clsListLabel">
             <property name="text">
              <string>반</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clsSaveBtn">
          <property name="text">
           <string>저장</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="subMngTab">
       <attribute name="title">
        <string>과목관리</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QFrame" name="frame_18">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="subSrhLabel">
             <property name="text">
              <string>조회된 과목 : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="subSrhStaLabel">
             <property name="text">
              <string>없음</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QFrame" name="frame_11">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>과목 리스트</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="subListWidget"/>
              </item>
              <item>
               <widget class="QFrame" name="frame_9">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="subSrhBtn">
                   <property name="text">
                    <string>조회</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="subDelBtn">
                   <property name="text">
                    <string>삭제</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="grdRngFrm">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QFrame" name="grdARngFrm">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="grdALabel">
                   <property name="text">
                    <string>A등급</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="grdAEdit1"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>~</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="grdAEdit2"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="grdBRngFrm">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="grdBLabel">
                   <property name="text">
                    <string>B등급</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="grdBEdit1"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>~</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="grdBEdit2"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="grdCRngFrm">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="grdCLabel">
                   <property name="text">
                    <string>C등급</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="grdCEdit1"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>~</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="grdCEdit2"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="subSaveBtn">
          <property name="text">
           <string>저장</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="subTitleLabel">
                <property name="text">
                 <string>과목명</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="subTitleEdit"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="grdAseWidget">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="currentIndex">
              <number>2</number>
             </property>
             <widget class="QWidget" name="grdAAseTab">
              <attribute name="title">
               <string>A등급</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QTableWidget" name="grdAAseList">
                 <column>
                  <property name="text">
                   <string>연번</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="grdBAseTab">
              <attribute name="title">
               <string>B등급</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QTableWidget" name="grdBAseList">
                 <column>
                  <property name="text">
                   <string>연번</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="grdCAseTab">
              <attribute name="title">
               <string>C등급</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QTableWidget" name="grdCAseList">
                 <column>
                  <property name="text">
                   <string>연번</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QFrame" name="frame_6">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QFrame" name="frame_7">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QPlainTextEdit" name="grdAseEdit">
                <property name="plainText">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="grdABtnFrm">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="grdAseAddBtn">
                <property name="text">
                 <string>추가</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="grdAseModBtn">
                <property name="text">
                 <string>수정</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="grdAseDelBtn">
                <property name="text">
                 <string>삭제</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ScrMngTab">
       <attribute name="title">
        <string>점수입력</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QFrame" name="frame_14">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QFrame" name="frame_16">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>과목선택</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_10"/>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>학급선택</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_9"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_16">
             <property name="text">
              <string>학급 불러오기</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_15">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>736</width>
                <height>616</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QTableWidget" name="tableWidget_2">
                 <row>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </row>
                 <column>
                  <property name="text">
                   <string>이름</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>학번</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>점수</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>평가</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_17">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QPushButton" name="pushButton_19">
                    <property name="text">
                     <string>평가 생성</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_18">
                    <property name="text">
                     <string>엑셀 출력</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_17">
                    <property name="text">
                     <string>저장</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
